src/release/main.d

```diff
@@ -726,14 +726,23 @@ ActionList prepareMajorRelease ( ref HTTPConnection con, ref Repository repo,
 
     auto current_branch = SemVerBranch(getCurrentBranch());
 
+    static bool branchExists ( SemVerBranch branch )
+    {
+        import std.process : executeShell;
```

Comment:
Please use `execute()` instead of `executeShell()` now! (`execute(["git", "rev-parse", "--verify", branch])`)

Reply:
You can't tell me what to do! You're not even my real mum!